test/index.js

```diff
@@ -579,6 +618,29 @@ describe('maybe', function() {
       assert(a.chain(a.of).equals(a));
     });
 
+    it('implements Traversable', function() {
+      var u = S.Nothing(),
+          C = Compose(Identity, S.Maybe),
+          lhs, rhs;
```

Comment:
I prefer one `var` keyword per declaration, though I don't mind grouping the uninitialized variables:

```javascript
var u = S.Nothing();
var C = Compose(Identity, S.Maybe);
var lhs, rhs;
```